### DISCUSSION TITLE
Request for 'Esc' input timeout at mod screen.

### DISCUSSION THREAD

**Discussion by ChaBoiNeil** [ppy/osu#19036] [Q&A]
### Type

Other

### Bug description

I use a keypad (simpad v2) and one of the buttons on it is an Escape key, and when I'm at the mods screen and press the button it sometimes seems to input the Escape input twice and it brings me to the main menu. This also happens when I exit the score results screen, and I get sent to the main menu. It can be annoying and I know it's a pretty unique "self-inflicted" issue, but I was wondering if it is possible to add a functionality that prevents this by adding a timer of like 50-100ms maybe. (It sends me to the main menu almost instantly so I don't think there would have to be a long wait period.)

If this is something that is just too out of your way and seen as picking out the tiniest quality of life issues that may not be present with everyone that's fine. Thought I would bring it up since this happens to me multiple times every time I play Lazer.

Thanks for taking the time to read this!

### Version

osu! 2022.704.0-lazer

### Logs

N/A

**Comment by peppy** [comment]
This sounds like a hardware issue. I'd consider returning it, or if the hardware is known faulty, check the drivers for a debounce / anti-jitter setting.

**Reply by ChaBoiNeil** [reply]
It's a HID that is basically a 2-key keyboard so there are no drivers for it. All I know is that I don't have this problem in stable so I figured it might be Lazer since that's the only version I have this issue with.

**Reply by peppy** [reply]
The issue is that lazer is pulling for input at a higher rate, I guess. We may be able to offer polling rate selection as an option.
